maybe/syscall_filters.py

```diff
@@ -95,6 +96,17 @@ def format_open(path, flags):
         return None
 
 
+def format_mknod(path, type):
```

Comment:
This method should be placed after `substitute_open` to maintain proper ordering